workflows/build.gradle

```diff
@@ -71,10 +71,15 @@ cordapp {
     workflow {
         name "WorkflowsModuleNameHere"
         versionId 1
-        vendor "VendorNameHere"
+        licence cordappLicense
+        vendor cordappVendorName
     }
 }
 
+// Use the name of the CorDapp as the name of the JAR file.
+// Set to the project directory name if the CorDapp name is not specified.
+archivesBaseName = cordapp.workflow.name.isPresent() ? cordapp.workflow.name.get() : projectDir.name
+
```

Comment:
This results in the cpb build with name `WorkflowsModuleNameHere-1.0-SNAPSHOT-package.cpb`
This then breaks the `3-buildCpis` task which is expecting to use value from gradle.properties > workflowsModule

Reply:
You can use the workflowsModule property directly like 

```
    workflow {
        name workflowsModule
        versionId 1
        licence cordappLicense
        vendor cordappVendorName
    }
```

And then maybe we should look at adding better comments to that property

```
# Specify the name of the workflows module
# This will be the name used to generate your CorDapp
workflowsModule=workflows
```

Reply:
Updated ✅